.env

```diff
@@ -0,0 +1,11 @@
+# dotenv .env file for environment variables 
+#
+# each developer should make their own .env.local file using this .env file
+# as a guide
+#
+# .env.local files are not checked into the remote repo
+#
+# reference these attributes in code like this:
+# config.fog_directory  = ENV['S3_BUCKET']
+
+S3_BUCKET=bogus-key-for-example
```

Comment:
You might consider removing this example to avoid any potential confusion in the future.